rubyforgood/human-essentials
# 3039-replace old partner text to new partner text

Resolves #3039 

### Description

Change old partner text to new partner text

### Type of change

* Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How Has This Been Tested?
- Go to partner page ( http://localhost:3000/diaper_bank/partners/2 ) 
- check labels
- edit partner Information

### Screenshot after change
![image](https://user-images.githubusercontent.com/836472/180578685-4a3dd006-ba46-413a-a72c-9f8467578423.png)

![image](https://user-images.githubusercontent.com/836472/181674655-56f8e93e-92ce-4749-8e0e-93227a594a84.png)


![image](https://user-images.githubusercontent.com/836472/181674732-9812f694-48cb-498e-9826-f6fca2f0df33.png)

![image](https://user-images.githubusercontent.com/836472/181674793-5ca7915d-9d1f-4fc9-b7d1-21417784e27b.png)
